src/main/java/teammates/ui/template/FeedbackResponseRow.java

```diff
@@ -22,8 +23,8 @@
     private String responseText;
     private List<FeedbackResponseCommentRow> responseComments;
 
-    public FeedbackResponseRow(int fbIndex, int personIndex, String personType,
-                               FeedbackResponseAttributes response, FeedbackSessionResultsBundle results) {
+    public FeedbackResponseRow(int fbIndex, int personIndex, String personType, FeedbackResponseAttributes response,
+                               FeedbackSessionResultsBundle results, Boolean showPcRow) {
```

Comment:
Do you need the new param to be the Object `Boolean` instead of the native `boolean`?

Reply:
Yes. It should be `boolean`, my bad